### PR TITLE
Check JS Best Practices

### PR DESCRIPTION
**Pull request by CollinsTatang**
Please verify check my code and let me know if do I follow the best JS practices

Thank you.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+      button.appendChild(img);
+      div.appendChild(inputCheckbox);
+      div.appendChild(inputTask);
+      div.appendChild(button);
+      li.appendChild(div);
+      list.appendChild(li);
+    });
+  }
+  const template = `
```
Comment (Mhdez221993) [inline src/index.js:118]: It is good practice to apply separation of concerns, mixing in HTML with your JavaScript is not ideal.

File: src/drag_drop.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/* --- This file contains the functions required to create the Drag&Drop effect    */
+let toLiIndex = null;
```
Comment (Mhdez221993) [inline src/drag_drop.js:2]: Global variables can cause naming collisions, security issues and slow your engine.

**Review by Mhdez221993 — COMMENTED** [review]
Well done!!

Your readme is descriptive, It is nice the way you apply your updateTasks method.

**Comment by CollinsTatang** [comment]
Hi Hernandez,
Thanks for all the changes requested.
I have done all the corrections you requested.
Thanks

**Review by Mhdez221993 — APPROVED** [review]
Well done @CollinsTatang your project is approved!!

**Pull request merged**